language/move-prover/src/spec_translator.rs

```diff
@@ -5,7 +5,7 @@
 
 use std::{
     cell::RefCell,
-    collections::{BTreeSet, HashMap},
+    collections::{hash_map, BTreeSet, HashMap},
```

Comment:
Ugh. Ugly we have to make this change twice. Perhaps better we add a test  `quantifiers.move` to tests/sources/functional and only do this, moving forward, for --v2. Like a number of other tests, if the test contains the line `flag: --v2` it will be run for v2 compilation scheme. We should be days away from having moved everything to v2. A new `quantifiers` test would anyway be good to have to test this for corner cases (e.g. including shadowing).

Reply:
Fair point. Let me add a test